### DISCUSSION TITLE
how to implment toast message in excalidraw

### DISCUSSION THREAD

**Discussion by Sohailamjad1** [excalidraw/excalidraw#5059] [Q&A] [answered]
Hi...can you come with one example like how to use set toast message in excalidraw...actually i need one confirmation box..this would be very helpful..Thanks

setToastMessage | (message: string) => void

**Comment by ad1992** [comment, marked as answer]
Please have a look at the docs, you can use [`ref`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#ref) to set the toast message.

**Reply by Sohailamjad1** [reply]
Thanks,...can you please check this one https://github.com/excalidraw/excalidraw/issues/5058
